languages/ruby/sig/projects_client.rbs

```diff
@@ -0,0 +1,9 @@
+class ProjectsClient
+  @command_runner: CommandRunner
+
+  def create_project: (project_name: String, organization_id: String) -> JSON
+  def delete_projects: (ids: Array[String]) -> JSON
+  def get: (project_id: String) -> JSON
+  def list_projects: (organization_id: String) -> JSON
+  def update_project: (id: String, project_put_request_name: String, organization_id: String) -> JSON
```

Comment:
We should use actual domain models if possible, `JSON` doesn't accurately describe what is returned.